e2e/brickmux_test.go

```diff
@@ -27,12 +27,11 @@ func TestBrickMux(t *testing.T) {
 	r.Nil(err)
 	r.NotNil(client)
 
-	// Turn on brick mux cluster option
 	optReq := api.ClusterOptionReq{
-		Options: map[string]string{"cluster.brick-multiplex": "on"},
+		Options: map[string]string{"cluster.brick-multiplex": "invalidValue"},
```

Comment:
so we don't have a test where we set brick multiplexing to off now?

Reply:
No, we don't have a test for this as of now but I will add it.